Assets/AppCenter/Plugins/AppCenterSDK/Auth/iOS/AuthInternal.cs

```diff
@@ -3,14 +3,38 @@
 
 #if UNITY_IOS && !UNITY_EDITOR
 using System;
-using System.Linq;
 using System.Collections.Generic;
 using System.Runtime.InteropServices;
+using Microsoft.AppCenter.Unity.Internal.Utility;
+using AOT;
 
 namespace Microsoft.AppCenter.Unity.Auth.Internal
 {
     class AuthInternal
     {
+
+#if ENABLE_IL2CPP
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+#endif
+        public delegate void NativeSignInCompletionHandler(IntPtr userInformation, IntPtr error);
+        private static AppCenterTask<UserInformation> _signInTask;
```

Comment:
Since in reality we don't have a use case for "queuing" due to the way the native SDK does it, might just be simpler to redo the single call check in unity. Would need to look at how the queue code would look like but seems like it would be over engineered solution.

Reply:
I agree with @guperrot. My suggestion is just do a null check on a `signInTask` before calling SignIn API and return exception "In progress" on a second call. I don't think we need a queue.